### ISSUE TITLE
iOS13

### ISSUE THREAD

**Issue by jesmrec** [owncloud/ios#1113]
Smoke tests performed over the app with iOS13:

- [X] Authentication basic
- [X] Authentication OAuth2
- [X] Multiaccount (switching account)
- [X] Passcode
- [X] Upload 50 files
- [X] Download 50 files
- [X] Instant uploads
- [X] Share with user/group
- [X] Share public
- [ ] Av. offline files
- [ ] Av. offline folders

**Comment by wolfgang-s** [comment]
Hi @jesmrec !

Did you test your changes on an iPad Pro 9.7 inch on iOS 13.1 ?

After login, in portrait mode you only see the ownCloud logo. That's it.

After switching to landscape mode, everything starts to work fine ... but portrait mode is unusable ... I can only see the detailsview.

I do not have time to look at it in detail, but comparing the console output, there are differences concerning the uisplitview and some layoutconstraint.

Cheers!

**Comment by jesmrec** [comment]
will take a look!

**Comment by jesmrec** [comment]
I've just tested the 3.8.2 version with an PadPro (iOS 13.1.3), and i can use the app in landscape mode. 

My questions are: did you perform a fresh install, or did you upgrade from the previous version? Any special setting in the app? which server version? (just in case the problem is caused by something related with sync)

thanks!

**Comment by wolfgang-s** [comment]
I did the following steps:
- Fresh install of XCode
- Fork
- Setup.MD Point 1 & 2
- Open in XCode Version 11.1 (11A1027) <- I think this is important, a customer of mine reported that it works when you compile using XCode 10
- Press run on iPad Pro 9.7 inch on iOS 13.1
- Connect to some demo ownCloud server
- After login only the logo is visible, the problem is the portrait mode, not landscape
- SourceTree show no changes to the source code

Basically the work around using XCode 10 is good enough for me ...

Cheers!

**Comment by jesmrec** [comment]
The legacy iOS app published in App Store is built under XCode10, and it is not adapted/tested under XCode11. This is the problem that causes the glitch you noticed. This app is legacy and the efforts of the team are focused in the new iOS app, that i recommend you to get:

https://github.com/owncloud/ios-app
https://apps.apple.com/app/id1359583808

afaik, it is not scheduled more effort to adapt the legacy app to the new Apple stuff. Am i worong @hosy @michaelstingl ?

**Comment by michaelstingl** [comment]
> afaik, it is not scheduled more effort to adapt the legacy app to the new Apple stuff. Am i wrong @hosy @michaelstingl ?

Yeah, the old app in App Store still works okay-ish. Only security issues will be fixed.

**Comment by taurushuang** [comment]
Hi. The current iOS i can’t access app of ownCloud9. Please help. Error is ‘Method not Allowed’

**Comment by taurushuang** [comment]
I can’t use new app, as it needs minimum ownCloud 10. And I don’t know how to transfer from owncloud 9 to 10

**Comment by michaelstingl** [comment]
@taurushuang please use the old app. Check the FAQ:
https://doc.owncloud.com/ios-app/ios_faq.html#compatibility-with-owncloud-server-older-than-version-10-0

**Comment by taurushuang** [comment]
@michaelstingl 
Thanks. But I’ve installed and login old version but when trying to access the folders this error pop up
![A6C578C7-4BEE-452A-A3BD-232D5C1ADCC4](https://user-images.githubusercontent.com/70319450/91426150-af2a3380-e88e-11ea-9690-fefa2da0be2d.jpeg)

**Comment by taurushuang** [comment]
I delete n reinstall app once again and it now works. Thanks

**Comment by michaelstingl** [comment]
Try to upgrade to a current oC version. oC 9 is EOL for quite long time. You can find detailed instructions in the docs.

**Comment by taurushuang** [comment]
I would love to. But do not dare to. Because I am not familiar with this at all and hardly understand the instructions. A friend previously setup cloud 9 for me. And we lost contact so I’m stuck in 9. The instructions look so complicated for some one with zero background on these - me.
